TransIT.API/Controllers/DataController.cs

```diff
@@ -110,5 +146,8 @@ public virtual async Task<IActionResult> Delete(int id)
             }
             return NoContent();
         }
+        
+        protected IEnumerable<TEntityDTO> EntityToDto(IEnumerable<TEntity> sequence) =>
+            sequence.Select(entity => _mapper.Map<TEntityDTO>(entity));
```

Comment:
`_mapper.Map<IEnumerable<TEntityDTO>>(sequence);`
I think this will be faster and more readable